SDL_LoadBMP.mediawiki

```diff
@@ -15,10 +14,31 @@ SDL_Surface* SDL_LoadBMP(const char* file)
 |}
 
 == Return Value ==
-<<Include(SDL_LoadBMP_RW, , , from="== Return Value ==", to="== Code Examples ==")>>
+Returns a pointer to a new [[SDL_Surface]] structure or NULL if there was an error; call [[SDL_GetError]]() for more information.
 
 == Remarks ==
-<<Include(SDL_LoadBMP_RW, , , from="== Remarks ==", to="== Related Functions ==")>>
+The new surface should be freed with [[SDL_FreeSurface]]().
+
+== Code Example ==
+
+<syntaxhighlight lang='c++'>
+// try to load the bitmap image
+const char * image_path = "path/to/image.bmp";
```

Comment:
Whoops! I've been working with C++ lately and forgot how to declare a c-string. This should be `const char image_path[] = "path/to/image.bmp";`